Installation Fails

Dependancies Installation fails. I've tried to install using a fresh Buster image on Rpi 4, 3 and Zero.  
Installation worked OK two weeks ago following the same process.

Sorry for the lack of info - I've been frustrated all day trying to do both a fresh installation and and update with a directory removal and installation.  The result always fails as shown below.

pi@raspberrypi:~ $ sudo apt-get install git
Reading package lists... Done
Building dependency tree       
Reading state information... Done
git is already the newest version (1:2.20.1-2+deb10u3).
The following package was automatically installed and is no longer required:
  python-colorzero
Use 'sudo apt autoremove' to remove it.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
pi@raspberrypi:~ $ git clone --recursive https://github.com/thomasjacquin/allsky.git
Cloning into 'allsky'...
remote: Enumerating objects: 2723, done.
remote: Counting objects: 100% (1160/1160), done.
remote: Compressing objects: 100% (644/644), done.
remote: Total 2723 (delta 732), reused 828 (delta 491), pack-reused 1563
Receiving objects: 100% (2723/2723), 57.42 MiB | 3.16 MiB/s, done.
Resolving deltas: 100% (1637/1637), done.
Submodule 'src/sunwait-src' (https://github.com/risacher/sunwait) registered for path 'src/sunwait-src'
Cloning into '/home/pi/allsky/src/sunwait-src'...
remote: Enumerating objects: 92, done.        
remote: Counting objects: 100% (43/43), done.        
remote: Compressing objects: 100% (40/40), done.        
remote: Total 92 (delta 18), reused 21 (delta 3), pack-reused 49        
Submodule path 'src/sunwait-src': checked out '102cb417ecbb7a3757ba9ee4b94d6db3225124c4'
pi@raspberrypi:~ $ cd allsky
pi@raspberrypi:~/allsky $ ./install.sh

**********************************************
*** Welcome to the Allsky Camera installer ***
**********************************************

* Dependencies installation

make[1]: Entering directory '/home/pi/allsky/src'
Package libusb-1.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libusb-1.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libusb-1.0' found
Makefile:46: *** Did not find USB Libraries, try 'make deps'.  Stop.
make[1]: Leaving directory '/home/pi/allsky/src'
make: *** [Makefile:30: deps] Error 2
Installing dependencies failed!
pi@raspberrypi:~/allsky $

--- a/src/Makefile
+++ b/src/Makefile
@@ -32,29 +32,10 @@ ifeq ($(PKGBUILD),)
   PKGBUILD=0
 endif
 
-deps:
-ifneq ($(shell id -u), 0)
-	@echo This must be ran with root permissions.
-	@echo Please run \'sudo make deps\'
-else
-	@echo `date +%F\ %R:%S` Installing build dependencies...
-	@apt update && apt -y install libopencv-dev libusb-dev libusb-1.0-0-dev ffmpeg gawk lftp jq imagemagick
-endif
-
-.PHONY : deps
-
-USB=$(shell pkg-config --cflags --libs libusb-1.0)
-ifeq (,$(USB))
-  $(error Did not find USB Libraries, try 'make deps')
-endif
-
+USB=$(shell pkg-config --exists libusb-1.0 && pkg-config --cflags --libs libusb-1.0)
 DEFS = -D_LIN -D_DEBUG -DGLIBC_20
 CFLAGS = -Wall -Wno-psabi -g -O2 -lpthread
 OPENCV = $(shell pkg-config --exists opencv && pkg-config --cflags --libs opencv || (pkg-config --exists opencv4 && pkg-config --cflags --libs opencv4))
-
-ifeq (,$(OPENCV))
-  $(error Did not find any OpenCV Libraries, try 'make deps')
-endif
 
 ifeq ($(platform), armv6l)
   CC = arm-linux-gnueabihf-g++
@@ -97,11 +78,31 @@ endif
 
 CFLAGS += $(DEFS) $(ZWOSDK)
 
-%:
-	@echo `date +%F\ %R:%S` nothing to do for $@
-
-all:capture capture_RPiHQ startrails keogram sunwait
+all:check_deps capture capture_RPiHQ startrails keogram sunwait
 .PHONY : all
+
+ifneq ($(shell id -u), 0)
+deps:
+	echo This must be ran with root permissions.
+	echo Please run 'sudo make deps'
+else
+deps:
+	@echo `date +%F\ %R:%S` Installing build dependencies...
+	@apt update && apt -y install libopencv-dev libusb-dev libusb-1.0-0-dev ffmpeg gawk lftp jq imagemagick
+endif
+
+.PHONY : deps
+
+
+check_deps:
+ifeq (,$(USB))
+	  $(error Did not find USB Libraries, try 'sudo make deps')
+endif
+ifeq (,$(OPENCV))
+	  $(error Did not find any OpenCV Libraries, try 'sudo make deps')
+endif
+.PHONY : check_deps
+
 
 sunwait:
 	@echo `date +%F\ %R:%S` Initializing sunwait submodule...
@@ -112,22 +113,22 @@ sunwait:
 	@cp sunwait-src/sunwait .
 	@echo `date +%F\ %R:%S` Done.
 
-capture:capture.cpp
+capture:check_deps capture.cpp
 	@echo `date +%F\ %R:%S` Building $@ program...
 	@$(CC)  $@.cpp -o $@ $(CFLAGS) $(OPENCV) -lASICamera2 $(USB)
 	@echo `date +%F\ %R:%S` Done.
 
-capture_RPiHQ:capture_RPiHQ.cpp
+capture_RPiHQ:check_deps capture_RPiHQ.cpp
 	@echo `date +%F\ %R:%S` Building $@ program...
 	@$(CC)  $@.cpp -o $@ $(CFLAGS) $(OPENCV)
 	@echo `date +%F\ %R:%S` Done.
 
-keogram:keogram.cpp
+keogram:check_deps keogram.cpp
 	@echo `date +%F\ %R:%S` Building $@ program...
 	@$(CC) $@.cpp -o $@ $(CFLAGS) $(OPENCV)
 	@echo `date +%F\ %R:%S` Done.
 
-startrails:startrails.cpp
+startrails:check_deps startrails.cpp
 	@echo `date +%F\ %R:%S` Building $@ program...
 	@$(CC) $@.cpp -o $@ $(CFLAGS) $(OPENCV)
 	@echo `date +%F\ %R:%S` Done.
@@ -186,3 +187,6 @@ clean:
 .PHONY : clean
 
 endif # Correct directory structure check
+
+%:
+	@echo `date +%F\ %R:%S` nothing to do for $@

